Project/Project/Views/Home/Privacy.cshtml

```diff
@@ -1,8 +1,47 @@
 ﻿@{
-    ViewData["Title"] = "Privacy Policy";
+ViewData["Title"] = "Info";
 }
 
-<h1>@ViewData["Title"]</h1>
+<div class="container mt-5">
+    <div class="impress mb-5">
+        <h1 class="mb-4 text-white">Impress</h1>
+        <div class="impress-card">
+            <h5>Project Title: PixelPrime</h5>
+            <p><strong>Created by:</strong><br>
+                Rui Felipe Pedrosso Ramos, Maximilian Horst August Huber, Amir Hassan, Marius Wenzel, Mehrulloh Boboev</p>
+            <p><strong>School:</strong><br>
+                ITS Stuttgart</p>
+            <p><strong>Class:</strong><br>
+                E2FI5</p>
+            <p><strong>Teacher:</strong><br>
+                Stephan Baldes</p>
+        </div>
+    </div>
 
-<p>Use this page to detail your site's privacy policy.</p>
+    <div class="information mb-5">
+        <h1 class="mb-4">Information</h1>
+        <p class="lead mb-5">This project was developed as part of the software engineering apprenticeship program at ITS Stuttgart. It represents the culmination of our second-year studies, showcasing the skills and knowledge we've acquired throughout our training.</p>
+    </div>
 
+    <div class="repo-card mb-5">
+        <div class="card bg-dark text-white">
+            <div class="card-body d-flex flex-column flex-sm-row align-items-center">
+                <div class="mb-3 mb-sm-0 me-sm-4">
+                    <img src="~/images/pixel_prime_logo.png" alt="PixelPrime Logo" class="img-fluid" style="max-width: 100px;">
+                </div>
+                <div>
+                    <h5 class="card-title">PixelPrime</h5>
+                    <h6 class="card-subtitle mb-2 text-muted">GitHub Repository</h6>
+                    <p class="card-text">Check out our project repository on GitHub for more details and source code.</p>
+                    @await Html.PartialAsync("_PrimaryButton",
+                    Tuple.Create(
+                    "View on GitHub",
+                    "style-1",
+                    "",
+                    "href=https://github.com/Maxiboy441/PixelPrime",
+                    "a"))
+                </div>
```

Comment:
Make the link open in a new tab. I think that can be done adding target="_blank"